driverkit/utils/generate

```diff
@@ -47,7 +47,7 @@ if [ -z ${TARGET_VERSION} ]; then
 fi
 
 ensure_kernelversion() {
-    TARGET_KERNEL_VERSION="$(expr match "${TARGET_KERNEL}" '^.*_\(.*\)$')"
+    TARGET_KERNEL_VERSION="$(expr match "${TARGET_KERNEL}" '^.*_\(\d*\)$')"
```

Comment:
This was broken for TARGET_KERNEL that contained a `x86_64` (`_64`)

Reply:
Good catch also here @FedeDP! Thanks.
It would be needed to test also if a valid version is kept, and not always being defaulted to "1".

Reply:
You were right; it wasn't working and always default to "1".
Now i take for granted that a kernelversion is always present in the crawler output (that holds true of course!).
This way we can actually simplify the code a bit.
Moreover, there is no simple way to distinguish between a "_%kv" and "x86_64" (when kv is missing), therefore the check would have need to be even more convoluted.

Reply:
If any kernelversion is not present, we can take it as a bug on the crawler.

Reply:
As discussed, I agree. I resolve this conversation :-)